src/axom/core/tests/utils_Timer.hpp

```diff
@@ -55,3 +55,39 @@ TEST(utils_Timer, timer_check_duration)
   EXPECT_GE(e, 1.0);
   EXPECT_LT(e, 2.0);
 }
+
+TEST(utils_Timer, timer_check_sum)
+{
+  const int N = 3;
+  axom::utilities::Timer t1(false);
+  axom::utilities::Timer t2(false);
+  for(int n = 0; n < N; ++n)
+  {
+    t2.start();
+    t1.start();
+    sleep(1);
+    t1.stop();
+    sleep(1);
+    t2.stop();
+  }
+
+  std::cout << "t1 measured: " << t1.summed() << "s in " << t1.cycleCount() << " cycles" << std::endl;
+  std::cout << "t2 measured: " << t2.summed() << "s in " << t2.cycleCount() << " cycles" << std::endl;
+
+  EXPECT_EQ(t1.cycleCount(), N);
+  EXPECT_EQ(t2.cycleCount(), N);
+
+  // Estimated inaccuracy per start-stop cycle.
+#ifdef _WIN32
+  const double tol = 0.01;
+#elif __OSX__
+  const double tol = 0.09;
+#else
+  const double tol = 0.0004;
+#endif
```

Comment:
where did these estimates come from?

Reply:
The CI test failures logs.  They're not scientific, but this we're just trying to catch potential errors in the Timer class, not high-resolution timings.